packages/itinerary-body/src/TransitLegBody/index.tsx

```diff
@@ -118,8 +120,13 @@ class TransitLegBody extends Component<Props, State> {
         ? transitOperator.logo
         : agencyBrandingUrl;
 
-    const expandAlerts =
-      alertsExpanded || (leg.alerts && leg.alerts.length < 3);
+    const maximumAlertCountToShowUncollapsed = 2;
```

Comment:
Declare this constant outside the class (e.g. above line 57).

Reply:
Done, thanks.